src/content/docs/get-started/release.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Release information
+description: Determine which versions of the drop-in components are compatible with each other and with Adobe Commerce.
+---
+
+Adobe organizes storefront application releases into logical suite versions and follow a regular release schedule (approximately once a month). However, Adobe may release updates as necessary.
+
+## Compatibility table
+
+The following table shows the compatibility between all of the storefront release items by suite version:
+
+| Release item | Version | |
+|---|---|---|
+| **Storefront suite** | 1.0.0 |
+| **Released** | December 2024 |
+| **Adobe Commerce** | 2.4.7 |
+| **Storefront compatibility package** | 4.7.0 |
+| **Drop-in SDK** | 0.37.0 |
+| **Drop-in components** | Product Details: 1.0.0<br/>Cart: 1.0.0<br/>Checkout: 1.0.0<br/>User Auth: 1.0.0<br/>User Account: 1.0.0<br/>Orders: 1.0.0 |
+
+## Code changes
+
+Every release includes changes to one or more codebases listed above. The scope of the change determines whether the MAJOR, MINOR, or PATCH number increases in the application's version according to [semantic versioning](https://semver.org/):
+
+> Given a version number MAJOR.MINOR.PATCH, increment the:
+> 
+> MAJOR version when you make incompatible API changes  
+> MINOR version when you add functionality in a backward compatible manner  
+> PATCH version when you make backward compatible bug fixes  
+
+Note: The "Storefront suite" version is more of a marketing construct and does not conform to semantic versioning.
```

Comment:
So...when we release a dropin and update `main` on boilerplate, should we expect to bump the suite version?

Reply:
No, not always. Package versions yes, and we'll update the package ranges